sdk/go/azuread/applicationPassword.go

```diff
@@ -151,15 +153,13 @@ func GetApplicationPassword(ctx *pulumi.Context,
 type applicationPasswordState struct {
 	// The resource ID of the application for which this password should be created. Changing this field forces a new resource to be created.
 	ApplicationId *string `pulumi:"applicationId"`
-	// The object ID of the application for which this password should be created
-	//
-	// Deprecated: The `applicationObjectId` property has been replaced with the `applicationId` property and will be removed in version 3.0 of the AzureAD provider
-	ApplicationObjectId *string `pulumi:"applicationObjectId"`
 	// A display name for the password. Changing this field forces a new resource to be created.
 	DisplayName *string `pulumi:"displayName"`
 	// The end date until which the password is valid, formatted as an RFC3339 date string (e.g. `2018-01-01T01:02:03Z`). Changing this field forces a new resource to be created.
 	EndDate *string `pulumi:"endDate"`
 	// A relative duration for which the password is valid until, for example `240h` (10 days) or `2400h30m`. Changing this field forces a new resource to be created.
+	//
+	// Deprecated: The `endDateRelative` property is deprecated and will be removed in a future version of the AzureAD provider. Please instead use the Terraform `timeadd()` function to calculate a value for the `endDate` property.
```

Comment:
Unfortunate comment for Pulumi users since there is no `timeadd`.

Reply:
So, after some digging, this value is actually coming from the deprecation notice in the code and not from the docs, so it's much harder to replace.

Reply:
On attempting to override this message via the ProviderInfo, it turns out that if the upstream provider has a deprecation message, that will be preferred over the locally provided string, so this is impossible to set.

Reply:
Bridged reference:
https://github.com/pulumi/pulumi-terraform-bridge/blob/f0fee4fb1db95e3aa8ebc505ab50c8152940c2d1/pkg/tfgen/generate.go#L636-L646